plugin/trino-kinesis/src/test/java/io/trino/plugin/kinesis/TestKinesisConfig.java

```diff
@@ -77,6 +79,7 @@ public void testExplicitPropertyMappings()
                 .setTableDescriptionLocation("/var/lib/kinesis")
                 .setDefaultSchema("kinesis")
                 .setHideInternalColumns(false)
+                .setTableDescriptionRefreshInterval(new Duration(1, TimeUnit.MINUTES))
```

Comment:
The value in `testExplicitPropertyMappings` should be different from the default value `1m`. 
https://github.com/trinodb/trino/pull/6521#discussion_r552339626